plugins/swagger-apibody.yml

```diff
@@ -0,0 +1,11 @@
+id: swagger-apibody
+name: Swagger ApiBody
+description: Add NestJS ApiBody Swagger decorator that specifies the structure of the request payload in a RESTful API for create (POST) and update (PUT/PATCH) actions.
+repo: amplication/plugins
```

Comment:
needs to be (to align with the other `.yml` files:

https://github.com/amplication/plugins/tree/master/plugins/swagger-apibody

Reply:
The `repo` key that is.

Reply:
why do we need 3 properties to hold the same value? `repo`, `github`, `website`

Reply:
Good question, remark was just to align it with the others.

Reply:
@levivannoort the existing three auth plugins use this style for the `repo` key already so while I think that aligning this across all plugins has value, it can be a separate PR and not a blocking comment for this PR.